src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -661,6 +665,16 @@ def load_arguments(self, _):
         c.argument('stdout', arg_type=bicep_stdout_type)
         c.argument('no_restore', arg_type=bicep_no_restore_type, help="When set, builds the bicep file without restoring external modules.")
 
+    with self.argument_context('bicep format') as c:
+        c.argument('file', arg_type=bicep_file_type, help="The path to the Bicep file to format in the file system.")
```

Comment:
> help="The path to the Bicep file to format in the file system."

Can we define help message into `bicep_file_type`, so that when reusing `bicep_file_type`, we can reduce the repeated definition of help message?